meta-ostro/conf/local.conf.sample

```diff
@@ -291,6 +291,18 @@ SUPPORTED_RECIPES_CHECK = "fatal"
 #
 #require conf/distro/include/ostro-os-production.inc
 
+# systemd-bootchart is a useful tool to analyze and optimize a system
+# boot time. The tool is available in Ostro and needs to be activated
+# by a kernel command-line parameter which requires to build a new
+# image. You also need to make sure that you enable CONFIG_SCHEDSTATS=y
+# (that will also set CONFIG_SCHED_INFO=y) in your kernel, this option
+# is not enabled by default. For instructions on how to modify an Ostro
+# kernel, please check doc/howtos/modifying-ostro-kernel.rst.
```

Comment:
I wasn't trying to render it, the problem is that it's not even created. It does get created if I run /lib/systemd/systemd-bootchart manually though... weird. I was wondering whether there could be some timing issue when the various parts of the filesystem get mounted but I'll take any other idea/recommendation.